packages/swap/src/event-handlers/v140/insufficientBoostLiquidity.ts

```diff
@@ -12,20 +13,38 @@ export const insufficientBoostLiquiditySchema = z.object({
 export const insufficientBoostLiquidity = async ({ prisma, event, block }: EventHandlerArgs) => {
   const { channelId, asset, amountAttempted } = insufficientBoostLiquiditySchema.parse(event.args);
 
-  const channel = await prisma.swapDepositChannel.findFirstOrThrow({
-    where: { channelId, srcAsset: asset },
+  const depositChannel = await prisma.depositChannel.findFirst({
+    where: { channelId, srcChain: assetConstants[asset].chain },
```

Comment:
```suggestion
    where: { channelId, srcChain: assetConstants[asset].chain, isSwapping: true },
```

Reply:
I wanted to still keep the assertion and check for an existing deposit channel disregarding the type of the channel